packages/sdk/base/src/identifier.ts

```diff
@@ -38,6 +38,7 @@ export enum IdentifierPrefix {
   DISCORD = 'discord',
   TELEGRAM = 'telegram',
   SIGNAL = 'signal',
+  GITHUB = 'github'
```

Comment:
maybe lint check is failing bc you need a comma here?

Reply:
added on a new commit